docs/tutorials/mobile.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Create a mobile game
+authors: 'Benjamin Fuentes'
```

Comment:
@timothymcmackin I saw a comment the other day that we don't add the names of individual contributors but the names of the entities to either docs/tutorials - I don't remember specifically. Will that be a problem here?

Reply:
@timothymcmackin Maybe also add Marigold and Nomadic Labs on the LICENSE file ?
The question is : why to put names or companies as authors. Is it to ease the maintainance finding the right people who wrote the article ?

Reply:
We are not consistent on whether the authors list the company names or the people's names, so for now do what feels correct to you. I don't know why we have that authors field; it's been there so I continue using it.

Reply:
IMHO, it is good for maintenance to keep authors list